pkg/util/capi/capi.go

```diff
@@ -5,13 +5,17 @@ package capi
 
```

Comment:
sort of strange having this in util.  Is this just used from cluster/driver?  Maybe rename existing capi folder in a follow up to oci and create new capi folder in cluster/driver with common code.

Reply:
Ok, I will address in a follow-up MR